packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3632,15 +3634,12 @@ export default class Meeting extends StatelessWebexPlugin {
           return Promise.resolve();
         }))
       .catch((error) => {
-        // Clean up stats analyzer, peer connection, and turn off listeners
+        // Reset stats analyzer and peer connection
         if (this.statsAnalyzer) {
           this.statsAnalyzer.stopAnalyzer();
-          this.statsAnalyzer = null;
-        }
-        if (this.mediaProperties.peerConnection) {
-          this.closePeerConnections();
```

Comment:
will need to check with allan on this ?

Reply:
everything that is red should be green and the green should be red.